scripts/wipe_sensitive_info_db.py

```diff
@@ -68,7 +35,7 @@ def main(argv):
     ):
       sys.exit(0)
```

Comment:
no need to sys.exit(0) inside a function, you can just return.

Reply:
OK, I thought it'd be more consistent with the `sys.exit(-1) above`

Reply:
Ok, if you are leveraging posix return codes this makes sense to me.

Reply:
The sys.exit(-1) was rendered unnecessary by moving to an all-flags solution so I changed that as well to match your suggestion, thanks!